.chronus/changes/tagMetadata-2024-9-23-16-55-56.md

```diff
@@ -0,0 +1,8 @@
+---
+changeKind: feature
+packages:
+  - "@typespec/openapi"
+  - "@typespec/openapi3"
+---
+
+a decorator for specify OpenAPI tag properties
```

Comment:
```suggestion
Add new `@tagMetadata` decorator to specify OpenAPI tag properties
```

Reply:
and it should have a different changelog for openapi3 library which has something like 
```
Add support for `@tagMetadata` decorator
```